staff/jakub-gaba/git/README.md

```diff
@@ -0,0 +1,18 @@
+git log - Shows commits
```

Comment:
When we are commenting a code is a good practice to comment on a line before the command itself, please change the file to follow this structure and make more readable:
// Comment about below line command (e.g. git status)
git status